frontend/src/components/task.jsx

```diff
@@ -1,29 +1,26 @@
 export default function Task(props) {
-    const { task, settasks } = props;
+    const { task, setTasks } = props;
 
-    const updatetask = async (taskId, taskStatus) => {
-        const res = await fetch(`backend/src/controllers/task.js${taskId}`, {
+    const updateTask = async (taskId, taskStatus) => {
+        await fetch(`http://localhost:3001/tasks/${taskId}`, {
             method: "PUT",
-            body: JSON.stringify({ status: taskStatus }),
+            body: JSON.stringify({ completed: taskStatus }),
```

Comment:
The field in schema to indicate whether the task is completed is `completed`, not `status`. So I changed this `status` by `completed` here and in other places below :)